src/main/java/frc/robot/InitBindings.java

```diff
@@ -58,7 +59,22 @@ public static void initDriveBindings(Drive drive) {
                 },
                 drive));
 
-    // pov right (reef 0-11 -> processor left -> processor right )
+    leftJoystick
+        .button(1)
+        .onTrue(
+            new InstantCommand(
+                () -> {
+                  drive.lockRotationToPosition(new Translation2d(3.5, 3));
```

Comment:
We shouldn't be passing in a magic position like this - the drivetrain subsystem should handle this itself based on a position target (making an enum with the only value being Reef would be a good starting point)

Reply:
we'll add to that as we go (coral station, net) so you could add those as well.

I would also like this function to be more named something more general than `lockRotationToPosition` - maybe `alignToFieldElement` would be good